snomed/com.b2international.snowowl.snomed.common/src/com/b2international/snowowl/snomed/common/SnomedTerminologyComponentConstants.java

```diff
@@ -57,6 +60,10 @@ private SnomedTerminologyComponentConstants() {}
 	public static final String SNOMED_URI_DEV = SNOMED_URI_BASE + "/xsct";
 	public static final String SNOMED_URI_ID = SNOMED_URI_BASE + "/id";
 	
+	public static final String SNOMED_CONCEPT_QUALIFIED_ID = TOOLING_ID + "." + SnomedConcept.TYPE;
```

Comment:
Move these constants to the SnomedReferenceSet class instead to avoid introducing a circular dependency between `snomed.datastore` and `snomed.common` modules.